worlds/dark_souls_3/__init__.py

```diff
@@ -985,13 +994,6 @@ def _add_npc_rules(self) -> None:
             "UG: Wolf Knight Leggings - shop after killing FK boss",
         ], self._has_any_scroll)
 
-        # Not really necessary but ensures players can decide which way to go
-        if self.options.enable_dlc:
-            self._add_entrance_rule(
-                "Painted World of Ariandel (After Contraption)",
-                self._has_any_scroll
-            )
```

Comment:
Shouldn't we add this back, since it's no longer part of the option?

Reply:
Since the scroll requirement is wrapped into `late_dlc` already, I figured it was fine to remove this. I thought it'd be strange/unintuitive if you could logically do PW1 but not PW2 when `late_dlc` is false.

Reply:
That's the intent of this—so that even without `late_dlc` players aren't _forced_ to avoid the catacombs/smouldering lake. `late_dlc` is only expected to open Irithyll or Castle, it shouldn't be necessary to open Catacombs.

Reply:
Added that back in now